qubesmanager/common_threads.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/python2
+#
+# The Qubes OS Project, http://www.qubes-os.org
+#
+# Copyright (C) 2011  Marek Marczykowski <marmarek@mimuw.edu.pl>
```

Comment:
Shouldn't this be you?

Reply:
Well, this file was thread_monitor.py while I was working on it and when I deleted thread_monitor class I renamed it.

Really I think that it could be removed moving the code to qube_manager.py and import from there. What do you think?

Reply:
> Well, this file was thread_monitor.py while I was working on it and when I deleted thread_monitor class I renamed it.

There was nothing else in thread_monitor.py, besides ThreadMonitor class, so this is really a new file.

> Well, this file was thread_monitor.py while I was working on it and when I deleted thread_monitor class I renamed it.

This would create circular dependency (qube_manager.py already imports settings.py), so better avoid it. This isn't a long file, but I think it still makes sense to keep it separate (maybe more such classes will be added there in the future).